generator.rb

```diff
@@ -0,0 +1,104 @@
+# from hw document, to find a vowel
+def is_vowel?(letter)
+  case letter
+    when "a", "e", "i", "o", "u", "A", "E", "I", "O", "U" 
+      return true
+    else 
+        return false
+    end
+end
+
+# determines if there is a vowel in a word
+def has_vowel(word)
+count = 0
+  word.each_char do |letter|
+    if is_vowel?(letter)
+        count += 1
+    else
+        count = count
```

Comment:
This line is unneeded! Count retains its value just fine without help. Also, if you're just looking for _any_ vowels, you can return after you find even one!